src/renderer/components/composer/menuAttachment.tsx

```diff
@@ -101,6 +105,36 @@ export default function MenuAttachment({
     tx,
   ])
 
+  const selectContact = async () => {
+    const { setLastPath } = rememberLastUsedPath(LastUsedSlot.Attachment)
```

Comment:
I don't get why you remember the last path here, makes no sense to me to open the file picker to the tmp directory when it's opened the next time.

Reply:
Just a leftover from copy paste. I removed it